src/types/index.ts

```diff
@@ -57,6 +57,7 @@ export interface ExerciseData {
 
 export type Answer = {
   id: ID;
+  question_id?: ID;
```

Comment:
Adding this because we will need a way to know which question the answer is for in a two step